packages/api/src/storage.ts

```diff
@@ -288,7 +292,15 @@ export default class Storage {
   }
 }
 
+/**
+ * @see schema.json
+ */
 interface Lockfile {
```

Comment:
maybe we just use `json-schema-to-ts` to generate this interface? 👀

Reply:
`json-schema-to-ts` doesn't seem to like it for some reason. might be a good thing to refactor at some point.

![Screen Shot 2023-10-19 at 12 20 48 PM](https://github.com/readmeio/api/assets/33762/6bb9c7e8-43a7-471a-a9ed-f50f577fbef6)

Reply:
talked offline and supplying a `.json` file to `json-schema-to-ts` is a TS limitation. https://github.com/ThomasAribart/json-schema-to-ts/blob/main/documentation/FAQs/does-json-schema-to-ts-work-on-json-file-schemas.md